cli/tests/pcluster/config/test_section_cluster.py

```diff
@@ -690,7 +690,13 @@ def test_cluster_section_from_file(mocker, config_parser_dict, expected_dict_par
         ("extra_json", None, {}, None),
         ("extra_json", "", {}, None),
         ("extra_json", "{}", {}, None),
-        ("extra_json", "{'test': 'test'}", {"test": "test"}, None),
+        ("extra_json", '{"test": "test"}', {"test": "test"}, None),
+        (
+            "extra_json",
+            "{'test': 'test'}",
+            {"test": "test"},
+            None,
+        ),  # WARNING it is considered a valid value by yaml.safe_load
```

Comment:
I'd remove the WARNING since in the internal representation we can consider it as valid, the important thing is to convert it to a valid JSON when converting to file or to cfn

Reply:
I'd keep this, because switching from yaml.safe_load to json.loads it will be considered invalid